ansible-ipi-install/group_vars/all.yml

```diff
@@ -40,3 +40,5 @@ worker_count: 0
 jumbo_mtu: false
 # set to true only if you requested a public routable VLAN for your cloud in scale lab
 routable_api: false
+# Lab dns, do not change
```

Comment:
@smalleni If it doesn't change often, lets move this to `roles/bootstrap/default/main.yml`